api/src/pcapi/core/finance/api.py

```diff
@@ -501,76 +505,102 @@ def _get_current_revenue(event: models.FinanceEvent) -> int:
 
 def _price_event(event: models.FinanceEvent) -> models.Pricing:
     new_revenue = _get_current_revenue(event)
-    is_incident_event = event.motive in (
-        models.FinanceEventMotive.INCIDENT_REVERSAL_OF_ORIGINAL_EVENT,
-        models.FinanceEventMotive.INCIDENT_NEW_PRICE,
-        models.FinanceEventMotive.INCIDENT_COMMERCIAL_GESTURE,
-    )
     individual_booking = event.bookingFinanceIncident.booking if event.bookingFinanceIncident else event.booking
     collective_booking = (
         event.bookingFinanceIncident.collectiveBooking if event.bookingFinanceIncident else event.collectiveBooking
     )
     booking = individual_booking or collective_booking
+    rule = None
+    amount = 0  # just to have a mypy-compliant initialization, it will get a real value afterwards
```

Comment:
si c'est juste pour mypy une initialisation pourrait être
```suggestion
    amount: int
```